src/components/PublishingTopbar/PublishingTopbar.tsx

```diff
@@ -18,53 +20,53 @@ import {
   StyledLink,
 } from './PublishingTopbar.style'
 
-type Channel = {
-  name: string
-  avatar: string
-}
 type ChannelInfoProps = {
   active?: boolean
-  member: string
-  channel: Channel
+  memberName?: string
+  channel: BasicChannelFieldsFragment | undefined
```

Comment:
If something can be `undefined` we usually use `?` notation: `channel?: BasicChannelFieldsFragment`

Reply:
More places like this in this file